pagopa/pn-frontend
# feat/PN-12538: fix bug on banner

## Short description
Fix condition to show appIO banner.

## List of changes proposed in this pull request
- Added IO disabled condition to hasCourtesyAddresses
- Re-ordered else condition to getDomicileData

## How to test
1. Login as PF
2. Make dod active and no address active
3. Check warning banner is active and not app-io banner
